### PR TITLE
Add Deck of cards to gain morale with a game of solitaire activity

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

```SUMMARY: Content "Add Deck of cards to gain morale with a game of solitaire activity"```

#### Purpose of change
Add Deck of cards to gain morale with a game of solitaire activity

#### Describe the solution
Just add the item - add a new iuse, and a new activity that gains morale, add to some relevant itemgroups, thats it really.

#### Describe alternatives you've considered
N/A

#### Testing
Loaded game, spawned cards, activated them, did activity, gained a bit of morale.

#### Additional context
N/A

### CODE REVIEW

**Comment by Funguss** [comment]
Can't wait to play poker with NPCs so they can win the shirt off my back!

**Comment by LaVeyanFiend** [comment]
> Can't wait to play poker with NPCs so they can win the shirt off my back!

Why let them win when you can pack the ballistic trump card in your sleeve?

File: data/json/items/fluff.json [hunk 1]
```diff
@@ -14,6 +14,21 @@
     "color": "white",
     "use_action": "MAGIC_8_BALL"
   },
+  {
+    "id": "deck_of_cards",
+    "type": "GENERIC",
+    "category": "other",
+    "name": "deck of cards",
```
Comment (Night-Pryanik) [inline data/json/items/fluff.json]: Plural name needed.

Comment (anothersimulacrum) [reply data/json/items/fluff.json]:
```suggestion
    "name": { "str": "deck of cards", "str_plural" "decks of cards" },
```

**Comment by Rail-Runner** [comment]
Could this be generalized to represent any kind of game? Like chess, checkers, or whatever else people could come up with.

**Comment by davidpwbrown** [comment]
> Could this be generalized to represent any kind of game? Like chess, checkers, or whatever else people could come up with.

I made the morale "MORALE_GAME" instead of "MORALE_VIDEO_GAME" so the text is more generic.

And thinking on it... yes I could make the iuse function generic "play game" and that means it could be re-used for other items that one could feasibly use to play a game solo.

Ill do that now.

**Comment by davidpwbrown** [comment]
There ive made the use action a generic "PLAY_GAME" that other stuff can use.

**Pull request merged**